src/firebase/authService.js

```diff
@@ -0,0 +1,48 @@
+import firebase from '@firebase/app';
+import '@firebase/auth';
+import userService from './userService';
+
+const authService = function() {
+    let auth = null;
+
+    function init() {
+        auth = firebase.auth();
+        auth.getRedirectResult().then(result => {
+            if(result && result.user) {
+                userService.addUser(result.user);
+            }
+        })
+
+        firebase.auth().onAuthStateChanged(user => {
+            if(user) {
+                console.log(user);
```

Comment:
Why are we doing this? Just for troubleshooting?

Reply:
My b